dms-core/src/main/kotlin/team/aliens/dms/domain/outing/spi/vo/OutingHistoryVO.kt

```diff
@@ -0,0 +1,13 @@
+package team.aliens.dms.domain.outing.spi.vo
+
+import java.time.LocalTime
+import java.util.UUID
+
+open class OutingHistoryVO(
+    val outingApplicationId: UUID,
+    val name: String?,
+    val outingType: String,
+    val companionCount: Int,

```

Comment:
```suggestion
    val outingCompanionCount: Int,

```
다른 곳에 있는 프로퍼티랑 이름 통일하면 좋을 것 같아요

Reply:
명세서만 보고 짜서 그런것 같습니다 ㅠㅠ
수정해두겠습니다